typedb/common/concurrent/scheduled_executor.py

```diff
@@ -0,0 +1,67 @@
+#
```

Comment:
Python's `scheduler` is pretty limited in what it can do. In order to make the Client code clearer, we need to introduce our own `ScheduledExecutor` class providing `schedule_at_fixed_rate`, akin to Java's `ScheduledThreadPoolExecutor`.